services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpointImpl.java

```diff
@@ -55,31 +55,62 @@
     private Queue executorQueue;
 
     @Override
+    public WindupExecution executeProjectWithContext(AnalysisContext analysisContext, Long projectId)
+    {
+        if (analysisContext == null)
+        {
+            analysisContext = this.analysisContextService.getDefaultProjectAnalysisContext(projectId);
+            if (analysisContext == null)
```

Comment:
Yes, the execution goes on with default config and the last execution's config becomes the default.